### ISSUE TITLE
Wrong numbers in remarks section of EM_SETSEL

### ISSUE THREAD

**Issue by maelh** [MicrosoftDocs/feedback#853]
**Link to article:**
https://docs.microsoft.com/en-us/windows/desktop/controls/em-setsel

**Problem:**
Somehow the minus sign was lost during a transition of the documentation system.

The remark section says:

> If the start is 0 and the end is 1, all the text in the edit control is selected. If the start is 1, any current selection is deselected.

But correct would be (as in the previous documentation):

> If the start is 0 and the end is –1, all the text in the edit control is selected. If the start is –1, any current selection is deselected.

**Comment by Powerhelmsman** [comment]
@ginami Can you follow up the issue?

**Comment by ginami** [comment]
Hi @maelh -- thank you for bringing this to our attention; we'll work on updating the page!

**Comment by BissetJ** [comment]
FYI,  I skimmed the other edit control messages and the these topics have the same -1 issue:
https://docs.microsoft.com/en-us/windows/desktop/controls/em-findtext
https://docs.microsoft.com/en-us/windows/desktop/controls/em-findtextex
https://docs.microsoft.com/en-us/windows/desktop/controls/em-findtextexw
https://docs.microsoft.com/en-us/windows/desktop/controls/em-findtextw
https://docs.microsoft.com/en-us/windows/desktop/controls/em-getsel
https://docs.microsoft.com/en-us/windows/desktop/controls/em-linefromchar
https://docs.microsoft.com/en-us/windows/desktop/controls/em-lineindex
https://docs.microsoft.com/en-us/windows/desktop/controls/em-linelength

And this one has the problem with other negative numbers (-1637 and other values in the Remarks section):
https://docs.microsoft.com/en-us/windows/desktop/controls/em-setfontsize

Note that looking at earlier versions of these docs, I believe that the missing negative signs we represented as EN DASH characters (Unicode 0x2013) in the older content.  Presumably, this character was lost in the translation into the docs.microsoft.com content.  

It appears that this issue is widespread (see also #1035).  **Rather than waiting for people to find them one at a time, search the original content for this character to find all the places where it will have been lost in translation.**

**Comment by BissetJ** [comment]
See also #465

**Comment by clandrew** [comment]
I agree with BissetJ, this looks to be a systemic issue rather than a few isolated instances. 
I don't have the right background to get into the wider issue, but as a temporary measure I merged a PR which fixes all the instances that were reported in #465, #1035 and this issue.
